reconstruction/cvt/src/main/java/org/jlab/rec/cvt/services/RecUtilities.java

```diff
@@ -927,9 +927,15 @@ Track recovTrkMisClusSearch(Seed seed, org.jlab.clas.tracking.trackrep.Helix hlx
                     c.getCluster2().setAssociatedTrackID(0);
                 }
             }
+            fittedTrack.setStatus(1); 
+        } else {
+            if(fittedTrack!=null) {
+                fittedTrack.setStatus(-1); 
+                if(Constants.getInstance().seedingDebugMode) 
+                    System.out.println("RECOVERED...with negative status "+fittedTrack.toString());
+                
+            }      
         }
         return fittedTrack;
     }
-
-    
 }
```

Comment:
In block of "else", track status is set for "if(fittedTrack!=null)", how about cases for fittedTrack==null?